src/graphql/typeDefs/mutations/registration.graphql

```diff
@@ -0,0 +1,8 @@
+type RegistrationMutation {
```

Comment:
I know it's above but don't we want to also add @auth to each of these mutations just in case?

Reply:
Sure we can